cmd/oras/internal/option/pusher.go

```diff
@@ -73,3 +93,22 @@ func decodeJSON(filename string, v interface{}) error {
 	defer file.Close()
 	return json.NewDecoder(file).Decode(v)
 }
+
+// getAnnotationsMap get resharp annotationslice to target type
+func getAnnotationsMap(ManifestAnnotationSlice []string, annotations map[string]map[string]string) error {
+	re := regexp.MustCompile(`=\s*`)
+	rawAnnotationsMap := make(map[string]string)
+	for _, rawAnnotation := range ManifestAnnotationSlice {
+		rawAnnotation := re.Split(rawAnnotation, 2)
```

Comment:
Dont reuse variable name for different types(`rawAnnotation` ), it's dangerous.

Reply:
fixed